101-media-services-create/azuredeploy.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "mediaServiceName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Media Services account. A Media Services account name is globally unique, all lowercase numbers or letters with no spaces."
+      }
+    }
+  },
+  "variables": {
+    "storageName": "[concat('storage', uniqueString(resourceGroup().id))]",
+    "storageType": "Standard_LRS"
+  },
+  "resources": [
+    {
+      "name": "[parameters('mediaServiceName')]",
+      "type": "Microsoft.Media/mediaServices",
+      "apiVersion": "2015-10-01",
+      "location": "[resourceGroup().location]",
+      "dependsOn": [
+        "[concat('microsoft.storage/storageaccounts/', variables('storageName'))]"
+      ],
+      "properties": {
+        "storageAccounts": [
+          {
+            "isPrimary": true,
+            "id": "[resourceId('microsoft.storage/storageaccounts/', variables('storageName'))]"
+          }
+        ]
+      }
+    },
+    {
+      "name": "[variables('storageName')]",
+      "type": "Microsoft.Storage/storageAccounts",
+      "apiVersion": "2015-06-15",
```

Comment:
Good point, so I changed it by "2016-01-01" like you said. I was curious to find where it's documented and I found it here: https://docs.microsoft.com/en-us/rest/api/storagerp/
But, I'm not sure this doc is up-to-date because when I run this PowerShell command `(Get-AzureRmResourceProvider -ProviderNamespace "Microsoft.Storage").ResourceTypes` I got: `ApiVersions      : {2016-12-01, 2016-05-01, 2016-01-01, 2015-06-15...}`. 
So it looks like the latest version is "2016-12-01", no?